jest.ts.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  globals: {
+    'ts-jest': {
+      babelConfig: 'babel.config.js',
+      tsConfig: './examples/using-typescript/tsconfig.json',
+    },
+  },
+  setupFilesAfterEnv: ['./scripts/testSetup.js'],
+  testURL: 'http://react-easy-state.com',
+  testRegex: '\\.test\\.tsx?$',
+  collectCoverage: true,
+  preset: 'ts-jest',
+  testEnvironment: 'jsdom',
+  coverageReporters: ['text'],
+  transform: {
+    '^.+\\.jsx?$': 'babel-jest', // Adding this line solved the issue
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  collectCoverageFrom: [
+    'src/**/*.{ts,tsx}',
+    'examples/**/*.{ts,tsx}',
+    '!examples/**/build/**',
```

Comment:
These lines are not needed (they are not included anyways). Feel free to keep them though.